### PR TITLE
Changed method names Nitro() in language and test files. #1581

### PR DESCRIPTION
**Pull request by 18-F-cali**
Changed the method names according to online problem description.

### CODE REVIEW

File: exercises/concept/need-for-speed/NeedForSpeed.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public void Drive()
         throw new NotImplementedException("Please implement the RemoteControlCar.Drive() method");
     }
 
-    public static RemoteControlCar TopModel()
+    public static RemoteControlCar Nitro()
     {
         throw new NotImplementedException("Please implement the (static) RemoteControlCar.TopModel() method");
```
Comment (ErikSchierboom) [inline exercises/concept/need-for-speed/NeedForSpeed.cs:24]:
```suggestion
        throw new NotImplementedException("Please implement the (static) RemoteControlCar.Nitro() method");
```

File: exercises/concept/need-for-speed/NeedForSpeedTests.cs [hunk 2]
```diff
@@ -87,29 +87,29 @@ public void Drive_until_battery_is_drained()
     [Fact(Skip = "Remove this Skip property to run this test")]
     public void Top_of_the_line_car_has_not_driven_any_distance()
```
Comment (ErikSchierboom) [inline exercises/concept/need-for-speed/NeedForSpeedTests.cs:88]:
```suggestion
    public void Nitro_car_has_not_driven_any_distance()
```

File: exercises/concept/need-for-speed/NeedForSpeedTests.cs [hunk 3]
```diff
@@ -87,29 +87,29 @@ public void Drive_until_battery_is_drained()
...
     {
-        var car = RemoteControlCar.TopOfTheLine();
+        var car = RemoteControlCar.Nitro();
         Assert.Equal(0, car.DistanceDriven());
     }
 
     [Fact(Skip = "Remove this Skip property to run this test")]
     public void Top_of_the_line_car_has_battery_not_drained()
```
Comment (ErikSchierboom) [inline exercises/concept/need-for-speed/NeedForSpeedTests.cs:95]:
```suggestion
    public void Nitro_car_has_battery_not_drained()
```

File: exercises/concept/need-for-speed/NeedForSpeedTests.cs [hunk 4]
```diff
@@ -87,29 +87,29 @@ public void Drive_until_battery_is_drained()
...
     {
-        var car = RemoteControlCar.TopOfTheLine();
+        var car = RemoteControlCar.Nitro();
         Assert.False(car.BatteryDrained());
     }
 
     [Fact(Skip = "Remove this Skip property to run this test")]
     public void Top_of_the_line_car_has_correct_speed()
```
Comment (ErikSchierboom) [inline exercises/concept/need-for-speed/NeedForSpeedTests.cs:102]:
```suggestion
    public void Nitro_car_has_correct_speed()
```

File: exercises/concept/need-for-speed/NeedForSpeedTests.cs [hunk 5]
```diff
@@ -87,29 +87,29 @@ public void Drive_until_battery_is_drained()
...
     {
-        var car = RemoteControlCar.TopOfTheLine();
+        var car = RemoteControlCar.Nitro();
         car.Drive();
         Assert.Equal(50, car.DistanceDriven());
     }
 
     [Fact(Skip = "Remove this Skip property to run this test")]
     public void Top_of_the_line_has_correct_battery_drain()
```
Comment (ErikSchierboom) [inline exercises/concept/need-for-speed/NeedForSpeedTests.cs:110]:
```suggestion
    public void Nitro_car_has_correct_battery_drain()
```

**Review by ErikSchierboom — COMMENTED** [review]
Great work! I've left some minor nits. 

Could you also update the https://github.com/exercism/csharp/blob/main/exercises/concept/need-for-speed/.meta/Exemplar.cs#L33 file?

**Comment by 18-F-cali** [comment]
@ErikSchierboom Made suggested Changes.

**Comment by ErikSchierboom** [comment]
> Could you also update the https://github.com/exercism/csharp/blob/main/exercises/concept/need-for-speed/.meta/Exemplar.cs#L33 file?

@18-F-cali Great! Could you also do the above and update the exemplar file?

**Comment by ErikSchierboom** [comment]
Closes https://github.com/exercism/csharp/issues/1581

**Comment by 18-F-cali** [comment]
Changed method name in .meta/Exemplar.cs

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
Merged. Thanks! 🎉

**Pull request merged**